### PR TITLE
[Cloud Security] Add support for GCP org 

### PR DESCRIPTION
**Pull request by orouz**
## What does this PR do?

Adds 2 new input vars: `organization_id` and `account_type`

closes https://github.com/elastic/integrations/issues/7398

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

### CODE REVIEW

File: packages/cloud_security_posture/data_stream/findings/manifest.yml [hunk 1]
```diff
@@ -123,6 +123,12 @@ streams:
     template_path: gcp.yml.hbs
     enabled: false
     vars:
+      - name: organization_id
+        type: text
+        title: Organization Id
+        multi: false
+        required: false
+        show_user: true
       - name: project_id
```
Comment (orouz) [inline packages/cloud_security_posture/data_stream/findings/manifest.yml:132]: the UI should indicate we support `project_id` or `organization_id`, not both.

**Review by maxcold — APPROVED** [review]

**Review by oren-zohar — COMMENTED** [review]
@orouz do we need to update the changelog and manifest version? this is for 8.11, right?

**Review by oren-zohar — CHANGES_REQUESTED** [review]
see my previous comment

File: packages/cloud_security_posture/data_stream/findings/agent/stream/gcp.yml.hbs [hunk 2]
```diff
@@ -4,11 +4,17 @@ config:
     deployment: gcp
     benchmark: cis_gcp
     gcp:
+      {{#if organization_id}}
+      organization_id: {{organization_id}}
+      {{/if}
+      {{#if project_id}}
       project_id: {{project_id}}
+      {{/if}
```
Comment (oren-zohar) [inline packages/cloud_security_posture/data_stream/findings/agent/stream/gcp.yml.hbs:12]:
```suggestion
      {{/if}}
```

**Review by oren-zohar — CHANGES_REQUESTED** [review]

File: packages/cloud_security_posture/data_stream/findings/agent/stream/gcp.yml.hbs [hunk 3]
```diff
@@ -4,7 +4,13 @@ config:
     deployment: gcp
     benchmark: cis_gcp
     gcp:
+      {{#if gcp.project_id}}
       project_id: {{gcp.project_id}}
+      {{/if}}
+      {{#if gcp.organization_id}}
+      organization_id: {{gcp.organization_id}}
+      {{/if}}
+      account_type: {{gcp.account_type}}
```
Comment (orouz) [inline packages/cloud_security_posture/data_stream/findings/agent/stream/gcp.yml.hbs:13]: `project_id` and `organization_id` depend on the value of `account_type`, so they may be missing. `account_type` is required.

File: packages/cloud_security_posture/changelog.yml [hunk 4]
```diff
@@ -4,6 +4,11 @@
 # 1.4.x - 8.9.x
 # 1.3.x - 8.8.x
 # 1.2.x - 8.7.x
+- version: "1.6.0"
```
Comment (orouz) [inline packages/cloud_security_posture/changelog.yml:7]: should this be a preview version?

Comment (oren-zohar) [reply packages/cloud_security_posture/changelog.yml:7]: yes

File: packages/cloud_security_posture/data_stream/findings/manifest.yml [hunk 5]
```diff
@@ -123,6 +123,18 @@ streams:
     template_path: gcp.yml.hbs
     enabled: false
     vars:
+      - name: gcp.account_type
+        type: text
+        title: Account Type
+        multi: false
+        required: false
```
Comment (orestisfl) [inline packages/cloud_security_posture/data_stream/findings/manifest.yml:130]: isn't this required?

Comment (orouz) [reply packages/cloud_security_posture/data_stream/findings/manifest.yml]: i thought our custom ui required us to not use it but this isn't the case for this var.

File: packages/cloud_security_posture/data_stream/findings/manifest.yml [hunk 6]
```diff
@@ -123,6 +123,18 @@ streams:
     template_path: gcp.yml.hbs
     enabled: false
     vars:
+      - name: gcp.account_type
+        type: text
+        title: Account Type
+        multi: false
+        required: true
+        show_user: true
```
Comment (uri-weisman) [inline packages/cloud_security_posture/data_stream/findings/manifest.yml:131]:
```suggestion
        show_user: false
```

**Review by uri-weisman — CHANGES_REQUESTED** [review]

**Review by uri-weisman — APPROVED** [review]

**Review by oren-zohar — APPROVED** [review]

File: packages/cloud_security_posture/data_stream/findings/agent/stream/gcp.yml.hbs (hunk 3 above)
Comment (kfirpeled) [reply packages/cloud_security_posture/data_stream/findings/agent/stream/gcp.yml.hbs:13]: @animehart we would need to have a migration here on fleet to add `account_type` when missing

Reference to fleet migrations that work on variable
https://github.com/elastic/kibana/tree/main/x-pack/plugins/fleet/server/saved_objects/migrations/security_solution

**Pull request merged**